### PR TITLE
RSDK-3314: Update Gripper to support GetGeometries

### PR DESCRIPTION
**Pull request by nthakur7**

### CODE REVIEW

File: components/gripper/fake/gripper.go [hunk 1]
```diff
@@ -59,3 +60,12 @@ func (g *Gripper) Stop(ctx context.Context, extra map[string]interface{}) error
 func (g *Gripper) IsMoving(ctx context.Context) (bool, error) {
 	return false, nil
 }
+
+// Geometries returns TODO
```
Comment (raybjork) [inline components/gripper/fake/gripper.go:64]: This comment exists several places.  Lets change it to something better

**Review by raybjork — CHANGES_REQUESTED** [review]
Looking great! I think with some test coverage we'll be there.  Should be enough to just add coverage to the fake gripper model

File: components/gripper/client.go [hunk 2]
```diff
@@ -5,13 +5,15 @@ import (
 	"context"
```
[the remark this answers is not in the document]
Comment (nthakur7) [reply components/gripper/client.go:5]: they are updated, are you not able to see it?

**Review by raybjork — APPROVED** [review]

File: components/gripper/client.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (biotinker) [reply components/gripper/client.go:5]: Oh I see. Weirdness with how github displays files. Never mind.

**Review by biotinker — APPROVED** [review]
LGTM

**Pull request merged**